lua/nvim-treesitter/highlight.lua

```diff
@@ -28,6 +28,7 @@ hlmap["character"] = "TSCharacter"
 hlmap["number"] = "TSNumber"
 hlmap["boolean"] = "TSBoolean"
 hlmap["float"] = "TSFloat"
+hlmap["attribute"] = "TSAttribute"
```

Comment:
Hmm.. We already have `field` and `property` what would be the difference with them ?

Reply:
I guess this kind of attribute is rather a code annotation. Like C++ attributes `[[maybe_unsed]]` or go annotation that tell you how specified field should be serialized. Maybe change to "@annotation"?

Reply:
I rather like `@annotation` as they will be used in java and rust too then.